inc/consent/namespace.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Altis\Core\Consent;
+
+use Altis;
+use Altis\Consent\Settings;
+
+/**
+ * Kick it off.
+ */
+function bootstrap() {
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\set_consent_defaults' );
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_plugins', 1 );
+}
+
+/**
+ * Save the defaults to the database if nothing has been set yet.
+ */
+function set_consent_defaults() {
+	$config  = Altis\get_config()['modules']['core']['consent'];
+	$options = get_option( 'cookie_consent_options' );
+
+	// Bail if we've turned consent off explicitly.
+	if ( $config === false ) {
```

Comment:
Should this be the same check as in `load_plugins()`? Say in some weird circumstance that `$config` was `null` this code would still execute but the plugins themselves wouldn't be loaded.
```suggestion
	if ( empty( $config ) ) {
```